kore/src/Kore/Attribute/Symbol/Anywhere.hs

```diff
@@ -0,0 +1,56 @@
+{- |
```

Comment:
Could you add this to `attributes.md`? I don't know what the `anywhere` attribute means, and, from its name, I can't figure out why it would have anything to do with injectivity.